### DISCUSSION TITLE
Broken link for "Metropolis font family" on uBO's about.html page

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2379] [General]
@gorhill,

I found a broken link on the [about.html](https://github.com/gorhill/uBlock/blob/master/src/about.html#L33) page of uBO. It seems the link to the "Metropolis font family" is broken. The [Metropolis font family repo](https://github.com/chrismsimpson/Metropolis) on [Chris Simpson's GitHub page](https://github.com/chrismsimpson) has been deleted. I searched his GitHub repo page and couldn't find it at all.

I would have already submitted a PR to fix the broken link in uBO; however, I have no idea what to change the link to or where due to the repo being deleted and turning up nothing but a dead end.

Also, with the repo deleted, I don't know if the "Metropolis font family" is still a thing or if it is still usable in uBO or other apps/extensions. I apologize that my knowledge of open-source software and licenses is lacking in this situation.

I am also happy to create an issue from this discussion if needed; I wasn't sure if it needed to be one.

**Comment by MasterKia** [comment]
https://github.com/uBlockOrigin/uBlock-issues/discussions/2269#discussioncomment-3672328

**Reply by ghost** [reply]
Thank you for the heads up and further information. Based on the [thread](https://discourse.mozilla.org/t/metropolis-font-repo-missing/69415/9) linked in the new repo, it seems like Chris Simpson was potentially threatened with legal action? Again, my knowledge of this legal, licensing, and whether or not this is open-source software is lacking. Is this font able to be used in uBO and other apps/extensions, or is it even worth worrying about/fighting potential trouble?

**Reply by ghost** [reply]
Also, if this is nothing to worry about, then so be it. I will submit a simple PR to update the link on uBO's about page. I will wait for @gorhill's input here before proceeding any further.

**Reply by MasterKia** [reply]
https://discourse.mozilla.org/t/metropolis-font-repo-missing/69415/10

> You certainly *can* use the release of ‘‘Clarity City’’, consistent with its OFL license, from GitHub, just like the company VMware does.
You can also use the ‘‘Metropolis’’ release, consistent with its OFL license, at FontsArena.com, just like the Mozilla Foundation does.

**Reply by ghost** [reply]
I saw that, but didn't know if the person who made that response was just some random person with zero knowledge or just guessing or if they were from Mozilla or something else.

**Reply by ghost** [reply]
I believe we are good with @gorhill's response. We can call it a day here.

**Comment by gorhill** [comment]
Let's link to <https://web.archive.org/web/20201013225635/https://github.com/chrismsimpson/Metropolis>?

**Reply by ghost** [reply]
PR created.

https://github.com/gorhill/uBlock/pull/3881
